test/unit_tests/test_ahs_device.py

```diff
@@ -208,6 +208,13 @@ def dummy_cfunc(t):
                             {"rampTime": 100e-9, "rabiCorrection": 1.00},
                         ],
                     },
+                    "rydbergLocal": {
```

Comment:
Is the rydbergLocal needed in the section above?

Similar to the schemas changes [here](https://github.com/amazon-braket/amazon-braket-schemas-python/pull/176/files#diff-081378977af8f39e04460fb427699ec2950b145130d76d199c59720023779d62R93-R101)